lib/account/account.controllers.js

```diff
@@ -17,29 +17,53 @@ module.exports = (models) => {
 
       return account;
     },
-    // getAccounts: async function(request, h) {
-    //   return models.Account.findAll()
-    //   .then(response => {
-    //     return response;
-    //   })
-    //   .catch(error => {
-    //     return error;
-    //   });
-    // },
+
+    getAccounts: async function(request, h) {
+      let returnAccounts;
+      try {
+        returnAccounts = await getAccounts();
+      } catch (e) {
+        throw Boom.badImplementation('Error during getAccounts(request.query). ' + e);
+      }
+      return h
+        .response(returnAccounts);
+    },
+
+    getAccount: async function(request, h) {
```

Comment:
If you could leave a TODO here to limit this route to superadmin or the user matching the id requested that'd be good.